dist/stat_data.py

```diff
@@ -409,19 +409,22 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     # Chunk cache statistics
     ##########################################
-    ChunkCacheStat('chunk_cache_bytes_inuse', 'total bytes used by the cache'),
-    ChunkCacheStat('chunk_cache_bytes_inuse_pinned', 'total bytes used by the cache for pinned chunks'),
-    ChunkCacheStat('chunk_cache_chunks_evicted', 'chunks evicted'),
-    ChunkCacheStat('chunk_cache_chunks_inuse', 'total chunks held by the chunk cache'),
-    ChunkCacheStat('chunk_cache_chunks_pinned', 'total pinned chunks held by the chunk cache'),
-    ChunkCacheStat('chunk_cache_exceeded_capacity', 'could not allocate due to exceeding capacity'),
-    ChunkCacheStat('chunk_cache_io_failed', 'number of times a read from storage failed'),
-    ChunkCacheStat('chunk_cache_lookups', 'lookups'),
-    ChunkCacheStat('chunk_cache_misses', 'number of misses'),
-    ChunkCacheStat('chunk_cache_chunks_loaded_from_flushed_tables', 'number of chunks loaded from flushed tables in chunk cache'),
-    ChunkCacheStat('chunk_cache_retries', 'retried accessing a chunk while I/O was in progress'),
-    ChunkCacheStat('chunk_cache_spans_chunks_read', 'aggregate number of spanned chunks on read'),
-    ChunkCacheStat('chunk_cache_toomany_retries', 'timed out due to too many retries'),
+    ChunkCacheStat('chunkcache_bytes_inuse', 'total bytes used by the cache'),
+    ChunkCacheStat('chunkcache_bytes_inuse_pinned', 'total bytes used by the cache for pinned chunks'),
+    ChunkCacheStat('chunkcache_chunks_evicted', 'chunks evicted'),
+    ChunkCacheStat('chunkcache_chunks_inuse', 'total chunks held by the chunk cache'),
+    ChunkCacheStat('chunkcache_chunks_loaded_from_flushed_tables', 'number of chunks loaded from flushed tables in chunk cache'),
+    ChunkCacheStat('chunkcache_chunks_pinned', 'total pinned chunks held by the chunk cache'),
+    ChunkCacheStat('chunkcache_created_from_metadata', 'total number of chunks created, and linked to their corresponding on-disk offsets using information extracted from metadata.'),
```

Comment:
I feel like this is unnecessarily wordy for a stat description - perhaps shortening "` offsets using information extracted from metadata`" to "`using the metadata`."

Reply:
That's a good call. I'd probably go further - it's not clear *why* we create these from the metadata, so how about `total number of chunks inserted on startup from persisted metadata`?

Reply:
I like that better actually - I wasn't sure how much detail we were willing to let go given the original length.

Reply:
Yes, that makes sense. I will update it, thanks!

Reply:
updated in [fee50db](https://github.com/wiredtiger/wiredtiger/pull/9759/commits/fee50dbca462331124dbe70417282bd2e933c7de)